frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/inline_page/inline_page_reference.dart

```diff
@@ -107,7 +108,7 @@ class InlinePageReferenceService {
     for (final view in views) {
       final SelectionMenuItem pageSelectionMenuItem = SelectionMenuItem(
         icon: (editorState, isSelected, style) => SelectableSvgWidget(
-          name: view.iconName,
+          data: FlowySvgData(view.iconName),
```

Comment:
I would've preferred not to do this, but in some areas, the user's selected emoji is stored as the asset path somewhere in the database. Hence, it's better to just construct from the path rather than the constant generated in `FlowySvgs`